plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/transition/DefaultViewportTransitionImpl.kt

```diff
@@ -49,9 +54,17 @@ internal class DefaultViewportTransitionImpl(
     completionListener: CompletionListener
   ): Cancelable {
     var isCancelableCalled = false
+    var animatorSet: AnimatorSet? = null
+    val startCamera = cameraDelegate.cameraState
+    val completedChildAnimators = mutableSetOf<CameraAnimatorType>()
+    var keepObserving = true
     val cancelable = to.observeDataSource { cameraOptions ->
-      startAnimation(
-        createAnimatorSet(cameraOptions, options.maxDurationMs)
+      animatorSet?.updateTargetCameraOptions(
+        completedChildAnimators,
+        startCamera,
+        cameraOptions
+      ) ?: kotlin.run {
```

Comment:
`kotlin.run`? something new to me

Reply:
Because we have a method `run` in this class, so need to specify we are using `kotlin.run` here.